src/fonts-page-layout/fonts-page-layout.css

```diff
@@ -0,0 +1,18 @@
+.fonts-page-layout {
+	display: grid;
+	grid-template-columns: 1fr 250px;
+	margin: 0;
+	padding: 0;
+}
+
+.fonts-page-layout > main {
+	padding: 2rem;
+}
+
+#wpfooter {
+	display: none;
```

Comment:
why hide the footer?

Reply:
To avoid overlapping with the fonts sidebar.